File: examples/java-calculator-testng/src/test/java/io/cucumber/examples/testng/RunCucumberTest.java
package io.cucumber.examples.testng;

import io.cucumber.testng.AbstractTestNGCucumberTests;
import io.cucumber.testng.CucumberOptions;
import org.testng.annotations.DataProvider;

@CucumberOptions(plugin = { "html:target/results.html", "message:target/results.ndjson" })
public class RunCucumberTest extends AbstractTestNGCucumberTests {

    @DataProvider(parallel = true)
    @Override
    public Object[][] scenarios() {
        return super.scenarios();
    }

}

File: examples/calculator-kotlin-testng/src/test/kotlin/io/cucumber/examples/calculator/RunCucumberTest.kt
package io.cucumber.examples.calculator

import io.cucumber.testng.AbstractTestNGCucumberTests
import io.cucumber.testng.CucumberOptions
import org.testng.annotations.DataProvider


@CucumberOptions(
    plugin = ["html:target/results.html"],
//    tags = "@shopping"
)
class RunCucumberTest : AbstractTestNGCucumberTests() {

    @DataProvider
    override fun scenarios(): Array<Array<Any>> {
        return super.scenarios()
    }

}
